devenv/setupRHELonZ.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# To get started:
```

Comment:
Could you add a comment explaining why this file exists? Something like "Development on Z is done on the native OS, not in Vagrant. This script can be used to set things up in RHEL on Z, similar to devenv/setup.sh which does the same for Vagrant. See https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/install.md"